src/utils/artifacts/DapiServer.json

```diff
@@ -0,0 +1,1468 @@
+{
```

Comment:
We probably will need an abi directory as well, but if we are going to deploy and have all the information in this repository, I think this will be redundant